app/lib/frontend/handlers/package.dart

```diff
@@ -264,6 +270,7 @@ Future<shelf.Response> packageAdminHandler(
     packageName: packageName,
     versionName: null,
     assetKind: null,
+    loadUploaderEmails: true,
```

Comment:
Since this is the only place where we load uploader emails, why not remove it from `PackagePageData` and load here, and then pass it explicitly to `renderPkgAdminPage`?

Reply:
Good point, I'll create a different PR for that.